acceptance/apps_test.go

```diff
@@ -498,8 +488,7 @@ configuration:
 
 				By("pushing the app specified in the manifest")
 
-				out, err := env.Epinio("", "apps", "push", manifestPath)
-				env.OnStageFailureShowStagingLogs(err, out, appName)
+				out, err := env.EpinioPush("", appName, manifestPath)
```

Comment:
I guess this case is why we have to repeat the `--name appName` in all the other cases, instead of having `Push` adding it automatically to the command line, right ?

Reply:
Yes, the app name arg is now duplicated in most requests and it's not really apparent why, unless one looks into the EpinioPush func. However it does shorten the code nicely, at least around pushing, the other epinio commands are still verbose.

I also tried this https://github.com/epinio/epinio/compare/dedup-staginglogs-handler-alt4. Doesn't read as nice, but maybe less surprising?